lighthouse-core/runner.js

```diff
@@ -144,7 +144,7 @@ class Runner {
   static getAuditList() {
     return fs
         .readdirSync(path.join(__dirname, './audits'))
-        .filter(f => /\.js$/.test(f));
+        .filter(f => /\.js$/.test(f) && f !== 'audit.js');
```

Comment:
`getAuditList` and `getGathererList` were incorrectly including the `Audit` and `Gatherer` base classes